sbnobj/Common/POTAccounting/classes_def.xml

```diff
@@ -5,13 +5,22 @@
    <version ClassVersion="11" checksum="3354055579"/>
    <version ClassVersion="10" checksum="13541"/>
   </class>
+  <class name="sbn::NuMISpillInfo" ClassVersion="13" >
```

Comment:
This is a new object, correct? If so, I think that it'd be better to change ClassVersion13 to 10, and remove the older ones. No need to keep versioning around for what you used as you developed.